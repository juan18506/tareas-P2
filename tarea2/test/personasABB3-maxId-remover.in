crearArbolPersonas

crearAgenda
crearPersona 5 55 Persona5
crearFecha 5/55/55
crearEvento 5 Descripcion5
agregarEventoPersona
insertarPersonaArbolPersonas

# arbol con 1 elemento - maxId 5
maxIdPersonaArbolPersonas

crearAgenda
crearPersona 10 100 Persona10
insertarPersonaArbolPersonas

crearAgenda
crearPersona 2 22 Persona2
insertarPersonaArbolPersonas

# arbol con 3 elementos - maxId 10
maxIdPersonaArbolPersonas

crearAgenda
crearPersona 20 200 Persona20
insertarPersonaArbolPersonas

crearAgenda
crearPersona 7 77 Persona7
insertarPersonaArbolPersonas

crearAgenda
crearPersona 1 11 Persona1
insertarPersonaArbolPersonas

crearAgenda
crearPersona 3 33 Persona3
insertarPersonaArbolPersonas

# arbol con 7 elementos - maxId 20
maxIdPersonaArbolPersonas

# arbol con 7 elementos
imprimirArbolPersonas

# remuevo la persona 20
removerArbolPersonas 20

imprimirArbolPersonas

# remuevo la persona 5
removerArbolPersonas 5

imprimirArbolPersonas

# remuevo la persona 2
removerArbolPersonas 2

# arbol con 4 elementos - maxId 10
maxIdPersonaArbolPersonas

crearAgenda
crearPersona 4 44 Persona4
# inserto la persona 4
insertarPersonaArbolPersonas

imprimirCantidadArbolPersonas
imprimirArbolPersonas

liberarArbolPersonas

Fin
