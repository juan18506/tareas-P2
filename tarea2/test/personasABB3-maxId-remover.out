1>2>3>4>5>6>7>8>#  arbol con 1 elemento - maxId 5.
9>La persona con mayor id en el árbol es: 
Persona 5:  Persona5, 55 años
Evento 5:  Descripcion5
Fecha: 5/55/55
10>11>12>13>14>15>16>#  arbol con 3 elementos - maxId 10.
17>La persona con mayor id en el árbol es: 
Persona 10:  Persona10, 100 años
18>19>20>21>22>23>24>25>26>27>28>29>30>#  arbol con 7 elementos - maxId 20.
31>La persona con mayor id en el árbol es: 
Persona 20:  Persona20, 200 años
32>#  arbol con 7 elementos.
33>Persona 1:  Persona1, 11 años
Persona 2:  Persona2, 22 años
Persona 3:  Persona3, 33 años
Persona 5:  Persona5, 55 años
Evento 5:  Descripcion5
Fecha: 5/55/55
Persona 7:  Persona7, 77 años
Persona 10:  Persona10, 100 años
Persona 20:  Persona20, 200 años
34>#  remuevo la persona 20.
35>36>Persona 1:  Persona1, 11 años
Persona 2:  Persona2, 22 años
Persona 3:  Persona3, 33 años
Persona 5:  Persona5, 55 años
Evento 5:  Descripcion5
Fecha: 5/55/55
Persona 7:  Persona7, 77 años
Persona 10:  Persona10, 100 años
37>#  remuevo la persona 5.
38>39>Persona 1:  Persona1, 11 años
Persona 2:  Persona2, 22 años
Persona 3:  Persona3, 33 años
Persona 7:  Persona7, 77 años
Persona 10:  Persona10, 100 años
40>#  remuevo la persona 2.
41>42>#  arbol con 4 elementos - maxId 10.
43>La persona con mayor id en el árbol es: 
Persona 10:  Persona10, 100 años
44>45>46>#  inserto la persona 4.
47>48>La cantidad de personas en el árbol es: 5.
49>Persona 1:  Persona1, 11 años
Persona 3:  Persona3, 33 años
Persona 4:  Persona4, 44 años
Persona 7:  Persona7, 77 años
Persona 10:  Persona10, 100 años
50>51>Fin.
